db/schema_tables.cc

```diff
@@ -1566,7 +1599,7 @@ static std::vector<data_type> read_arg_types(replica::database& db, const query:
     }
 #endif
 
-static shared_ptr<cql3::functions::user_function> create_func(replica::database& db, const query::result_set_row& row) {
+static shared_ptr<cql3::functions::user_function> create_func(replica::database& db, const query::result_set_row& row, lw_shared_ptr<query::result_set>) {
```

Comment:
We now have two result-set style rows. How can we distinguish between the two? They should have names that describe their roles.

Also, one is a ow and another is a result set, which is odd.

Reply:
And that parameter is not used.

Reply:
Removed this argument (because of before/after schema)